builtin/logical/pki/cert_util.go

```diff
@@ -670,7 +673,7 @@ func generateCert(ctx context.Context,
 
 		if data.SigningBundle == nil {
 			// Generating a self-signed root certificate
-			entries, err := getURLs(ctx, input.req)
+			entries, err := getURLs(ctx, input.req.Storage)
```

Comment:
It is somewhat unfortunate that both `storageContext` and `inputBundle` have the storage object. Perhaps use `sc.Storage` instead? 

If you are feeling cautious or pedantic, then maybe you should add a guard to functions that take these two objects to verify that their storage object is then same?